### PR TITLE
Update CV and about section

### PR DESCRIPTION
**Pull request by caroleolivier**
@simonproffitt : can you review where I put a comment below pls? (checking my English basically :))

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -10,11 +10,14 @@ <h1 class="hi">
                 Hi, I am Carole
             </h1>
             <p>
-                I am a programmer. I live in London.
+                I am a software engineer. I live in London.
```
Comment (caroleolivier) [inline about/index.html:13]: this file pls!

File: cv/experience/bulb.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### Software Engineer  *December 2017 – Present*
```
Comment (caroleolivier) [inline cv/experience/bulb.md]: this file too pls!

File: cv/main-view/career-profile.html [hunk 3]
```diff
@@ -1,10 +1,7 @@
 <section class="section summary-section">
     <h2 class="section-title"><i class="fa fa-user"></i>Career Profile</h2>
     <div class="summary">
-        I am a software engineer with 7 years experience in the financial sector.
-        I have been working as a full stack developer with the majority of my time spent developing UIs and managing clients requirements.
-        Therefore, I have become very pragmatic and understand the importance of getting things done for clients and/or a business.
-        I am also passionate, love my work and care about my colleagues that's why
-        I hope my next move will be to a company with a clear mission and where growing and sharing are important company values.
+        I am a fullstack developer at Bulb. We are changing the energy industry for the better, making it simpler, cheaper, greener.
```
Comment (caroleolivier) [inline cv/main-view/career-profile.html:4]: and finally here, thanks dude.

**Review by simonproffitt — CHANGES_REQUESTED** [review]
Have added a couple of comments, then good to go

**Review by simonproffitt — APPROVED** [review]

**Pull request merged**